src/jsifier.js

```diff
@@ -197,13 +197,14 @@ function ${name}(${args}) {
         return '';
       }
 
-      let noExport = false;
+      let isStub = false;
```

Comment:
What does "stub" mean here? ("export" before is more obvious, but this might be better if it's more accurate, even if the word is less immediately obvious. But if so, maybe a comment?)